cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -88,6 +90,14 @@ func newASGCache(service autoScalingWrapper, explicitSpecs []string, autoDiscove
 	return registry, nil
 }
 
+var getInstanceTypeForAsg = func(m *asgCache, asg *asg) (string, error) {
```

Comment:
nit: This function is not actually getting instance type from ASG.
getInstanceTypeForAsgFromCache?